projects/span_finder/configs/span_finder/config_trf.cfg

```diff
@@ -0,0 +1,177 @@
+[paths]
+train = null
+dev = null
+vectors = null
+init_tok2vec = null
+
+[vars]
+spans_key = null
+
+[system]
+gpu_allocator = "pytorch"
+seed = 0
+
+[nlp]
+lang = "en"
+pipeline = ["transformer","span_finder","spancat"]
+batch_size = 128
+disabled = []
+before_creation = null
+after_creation = null
+after_pipeline_creation = null
+tokenizer = {"@tokenizers":"spacy.Tokenizer.v1"}
+
+[components]
+
+[components.transformer]
+factory = "transformer"
+max_batch_items = 4096
+set_extra_annotations = {"@annotation_setters":"spacy-transformers.null_annotation_setter.v1"}
+
+[components.transformer.model]
+@architectures = "spacy-transformers.TransformerModel.v1"
+name = "roberta-base"
+
+[components.transformer.model.get_spans]
+@span_getters = "spacy-transformers.strided_spans.v1"
+window = 128
+stride = 96
+
+[components.transformer.model.tokenizer_config]
+use_fast = true
+
+[components.span_finder]
+factory = "experimental_span_finder"
+threshold = 0.35
+predicted_key = "span_candidates"
+training_key = ${vars.spans_key}
+min_length = 0
+max_length = 0
+
+[components.span_finder.scorer]
+@scorers = "spacy-experimental.span_finder_scorer.v1"
+predicted_key = ${components.span_finder.predicted_key}
+training_key = ${vars.spans_key}
+
+[components.span_finder.model]
+@architectures = "spacy-experimental.SpanFinder.v1"
+
+[components.span_finder.model.scorer]
+@layers = "spacy.LinearLogistic.v1"
+nO=2
+
+[components.span_finder.model.tok2vec]
+@architectures = "spacy-transformers.TransformerListener.v1"
+grad_factor = 1.0
+
+[components.span_finder.model.tok2vec.pooling]
+@layers = "reduce_mean.v1"
+
+[components.spancat]
+factory = "spancat"
+max_positive = null
+spans_key = ${vars.spans_key}
+threshold = 0.5
+
+[components.spancat.model]
+@architectures = "spacy.SpanCategorizer.v1"
+
+[components.spancat.model.reducer]
+@layers = "spacy.mean_max_reducer.v1"
+hidden_size = 128
+
+[components.spancat.model.scorer]
+@layers = "spacy.LinearLogistic.v1"
+nO = null
+nI = null
+
+[components.spancat.model.tok2vec]
+@architectures = "spacy-transformers.TransformerListener.v1"
+grad_factor = 1.0
+
+[components.spancat.model.tok2vec.pooling]
+@layers = "reduce_mean.v1"
+
+[components.spancat.suggester]
+@misc = "spacy-experimental.span_finder_suggester.v1"
+predicted_key = ${components.span_finder.predicted_key}
```

Comment:
```suggestion
candidates_key = ${components.span_finder.predicted_key}
```

Reply:
Ah, the setting in `span_finder_suggester` should probably still be called `candidates_key`? Since it's just reading in the candidates, not predicting/saving them to `doc.spans`.